dataplane/internal/kernel/genetlink.go

```diff
@@ -57,17 +58,23 @@ func NewGenetlinkPort(family, group string) (*GenetlinkPort, error) {
 	}, nil
 }
 
+type PacketMetadata struct {
+	SrcIfIndex int
+	DstIfIndex int
+	Context    int
```

Comment:
Just curious, what is Context? an int?

Reply:
just an extra value that the SAI client can configure